packages/boiler-task-selenium/src/runner/webdriverio/index.js

```diff
@@ -9,4 +9,11 @@ require('babel-register')({
 });
 require('babel-polyfill');
 
-exports.config = require('./config.js');
+const opts = {};
+if (process.argv[4] === '--tunnel') {
+  Object.assign(opts, {
+    maxInstances: 5,
```

Comment:
you can do `process.argv.indexOf('--tunnel') !== -1`

you could do the same if you decided to add a `--parallel` flag if local but running parallel browsers. see comments below in `spawn-process`